config/manager/manager.yaml

```diff
@@ -66,26 +66,6 @@ spec:
       protocol: TCP
       port: 8043
       targetPort: 8043
-    - name: metrics
-      protocol: TCP
-      port: 8080
-      targetPort: 8080
   selector:
     control-plane: controller-manager
   type: ClusterIP
----
-kind: ServiceMonitor
```

Comment:
Why is ServiceMonitor removed?

Reply:
this resource doesn't include in k8s, at least not by default.
attempt to use it with existing code fails. therefore it would be better removing it and adding it to the docs and later we can think on adding a rule to deploy metrics.